weave-js/src/components/RadixTooltip/Tooltip.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  TooltipContentProps,
+  TooltipPortalProps,
+  TooltipTriggerProps,
+} from '@radix-ui/react-tooltip';
+import React, {useState} from 'react';
+
+import * as RadixTooltip from './index';
+
+export type TooltipProps = {
+  trigger: React.ReactNode;
+  content: React.ReactNode | string;
+  tooltipProps?: RadixTooltip.TooltipProps;
+  triggerProps?: TooltipTriggerProps;
+  contentProps?: TooltipContentProps;
+  portalProps?: TooltipPortalProps;
+  /**
+   * This determines if the trigger and content components should be wrapped in
+   * RadixTooltip.Trigger and RadixTooltip.Content, respectively. Otherwise, it
+   * will just assume props passed in are properly wrapped in RadixTooltip anatomy.
+   * Having this option allows a bit more flexibilty and less prop passing.
+   *
+   * Setting it to false will tell the component to not apply the RadixTooltip.Trigger
+   * and RadixTooltip.Content around the props.
+   */
+  nowrap?: boolean;
+  removePortal?: boolean;
+};
+
+/**
+ * This is a more ergonomic Tooltip built on top of our RadixTooltip. If this component does not
+ * meet your needs, please refrain from modifying this and consider using RadixTooltip directly.
+ */
+export const Tooltip = ({
+  trigger,
+  content,
+  tooltipProps,
+  triggerProps,
+  contentProps,
+  portalProps,
```

Comment:
@connieelee is passing all these props excessive?

Reply:
at this point, why not just use RadixTooltip directly?

Reply:
i was thinking of this exact same thing.

Reply:
i'm inclined to remove all the pass-through props, and possibly even `nowrap` and `removePortal` because if the dev wants that level of flexibility they can just use the base RadixTooltip components instead of this "ergo" version